src/app/components/cds/slideout/Slideout.js

```diff
@@ -111,7 +111,7 @@ const Slideout = ({
 
 Slideout.defaultProps = {
   content: null,
-  footer: true,
+  footer: false,
```

Comment:
@sarahkeh somethign to get your eyes on specifically is this change to the slideout that I made. I noticed that if there is a slideout with no buttons or info, then the footer is still shown, just empty. So i figured its better to be opt-in instead of out.

Reply:
Also, does `footer` need to be its own boolean prop or instead showing the footer or not can/should be simply inferred from the existence of footer content?

Reply:
probably better answered/addressed by @sarahkeh as the author of that component, but seems reasonable

Reply:
It could probably be removed, my thinking was it could be easier in some instances where the slideout contains several buttons but might need to be hidden for certain users or in certain states to just set one value. But I'm not sure that's necessary, could definitely update it to just check if any of the buttons are there.

Reply:
Sounds good. That's more of a suggestion than a requirement. If it's going to be too much trouble, let it be